tests/unit/modules/election-lookup.js

```diff
@@ -108,7 +108,7 @@ describe('election lookup', function() {
     var titles = $rendered.map(function(idx, elm) {
       return $(elm).find('h3').text().trim();
     }).get();
-    expect(titles).to.deep.equal(['US President', 'NJ Senate', 'NJ House District 09']);
+    expect(titles).to.deep.equal(['US president', 'NJ Senate', 'NJ House District 09']);
```

Comment:
Technically for style, it should be lowercase `president`.  But I think you raise a really good point about consistency — the headers would be more jarring if they didn't look the same. I changed it back to the `President`.